packages/tessellate-request/.flowconfig

```diff
@@ -0,0 +1,17 @@
+[ignore]
+
+[include]
+
+[libs]
+./../../flow-typed/
```

Comment:
We should put flow typedefs into each package separately going forward because it makes the installation of required types easier and decouples packages more.

See tessellate-server for an example:

- [flow-type-install and flow-check scripts in package.json](https://github.com/zalando-incubator/tessellate/blob/master/packages/tessellate-server/package.json#L23)
-  [flow-typed and flow-bin dependencies in package.json](https://github.com/zalando-incubator/tessellate/blob/master/packages/tessellate-server/package.json#L45)
- [.flowconfig](https://github.com/zalando-incubator/tessellate/blob/master/packages/tessellate-server/.flowconfig#L6)

I didn't remove `[version]` from flowconfig in tessellate-server but I think it should probably be done if flow-bin is already declared in package.json.